api/user.go

```diff
@@ -228,3 +279,78 @@ func (ua *UserAPI) ChangePassword() {
 		ua.CustomAbort(http.StatusInternalServerError, "Internal error.")
 	}
 }
+
+// ToggleUserAdminRole handles Post api/users/{}/toggleadmin
+func (ua *UserAPI) ToggleUserAdminRole() {
```

Comment:
1)api should match the actual uri of the api
2)As for the implementation, it should be consistent with the project/:id/publicity, i.e. read the value of from the request.